basic_flutter/lib/presentation/widgets/profile_widget.dart

```diff
@@ -24,16 +25,16 @@ class ProfileWidget extends StatelessWidget {
     );
   }
 
-  Widget buildEditIcon(Color color, BuildContext context) => buildCircle(
+  Widget buildEditIcon(Color color, BuildContext context) => BuildCircle(
```

Comment:
This function isn't in use and can be deleted. If this is used later it should be a class.
Also you can add multiple classes in one file if the class is used only in this file.